apps/portlet-documents/src/main/webapp/vue-app/attachment/components/Attachment.vue

```diff
@@ -19,7 +19,8 @@
         :supported-documents="supportedDocuments"
         :attachments="attachments"
         :open-attachments-in-editor="openAttachmentsInEditor" />
-      <attachments-notification-alerts />
+      <attachments-notification-alerts
```

Comment:
Could you check if using the shared component `<exo-notification-alert />` solves the issue without adding the new CSS class ?